eslint.config.mjs

```diff
@@ -340,4 +340,17 @@ export default [
       'jsdoc/require-jsdoc': 'off',
     },
   },
+  {
+    files: [
+      'karma.conf.js',
+      'build/**/*.js',
+      'demo/**/*.js',
+      'lib/debug/asserts.js',
+      'lib/debug/log.js',
+      'test/**/**.js',
```

Comment:
Why?

Reply:
We often want to console log in tests, so they should be excluded.  I agree with this.